app/src/main/res/layout/vocable_confirmation_dialog.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="@dimen/dialog_width"
+    android:layout_height="wrap_content"
+    android:background="@android:color/white"
+    android:elevation="@dimen/dialog_shadow">
+
+    <TextView
+        android:id="@+id/dialog_title"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="@dimen/dialog_text_margin"
+        android:layout_marginTop="@dimen/dialog_text_margin"
+        android:layout_marginEnd="@dimen/dialog_text_margin"
+        android:textColor="@color/colorPrimaryDark"
+        android:textSize="@dimen/dialog_title_text_size"
+        android:textStyle="bold"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
+        tools:text="Leaving the app" />
+
+    <TextView
+        android:id="@+id/dialog_message"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="@dimen/dialog_text_margin"
+        android:layout_marginEnd="@dimen/dialog_text_margin"
+        android:textColor="@color/colorPrimaryDark"
+        android:textSize="@dimen/dialog_title_text_size"
```

Comment:
`dialog_title` and `dialog_message` share a bunch of common attributes - worth extracting to a style or do you see these two TextViews being the only use cases?